### PR TITLE
[SMP-1344] SMP 79230 0.6.0 release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
SMP 79230 0.6.0 release notes

https://harness.atlassian.net/wiki/spaces/PD/pages/21408317523/Release+Notes+Next+Gen-+06+15+2023
https://harness.atlassian.net/wiki/spaces/PD/pages/21408317504/Release+Notes+Current+Gen-+06+15+2023

For reviewers, previews are available:
[FG](https://6489eb1f07a29c08679cd992--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes)
[NG](https://6489eb1f07a29c08679cd992--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition)

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -14,7 +14,653 @@ import delete_project from './static/delete-project.png'
...
+- Harness updated the following images to use versioned tags: (SMP-1347)
+
+   - docker.io/harness/sto-plugin:1.12.0
+   - docker.io/curlimages/curl:8.1.1
+
+   The docker.io/harness/helm-init-container image will be updated to use a versioned tag in a future release.
+
+   Timescaledb-ha uses uses versioned tag pg13-ts2.9 (for pg13, Harness uses the TimescaleDB `ts2.9-oss-latest` image, which updates with any minor release and is currently 2.9.3).
+
+- The `PRUNE_KUBERNETES_RESOURCES` feature flag is now disabled by default in stages and Self-Managed Enterprise Edition releases.
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md:50]: stage cluster is internal detail 
so lets remove 'in stages and Self-Managed Enterprise Edition releases.'

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -14,7 +14,653 @@ import delete_project from './static/delete-project.png'
...
+```mdx-code-block
+<Tabs>
+  <TabItem value="What's new">
+```
+
+This release includes the following Harness module and component versions.
+
+| **Name** | **Version** |
+| :-- | :--: |
+| Helm Chart | [0.6.0](https://github.com/harness/helm-charts/releases/tag/harness-0.6.0) |
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md:28]: link does not work

Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md:28]: @akshay-khole This is a placeholder link and will be available when the helm chart is released.

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 3]
```diff
@@ -14,6 +14,76 @@ For Harness SaaS release notes, see [Harness SaaS Release Notes](https://develop
...
+
+#### Harness Delegate
+
+- Removed the `DELEGATE_IMAGE_TAG` version override when delegates with the immutable image type are enabled. (PL-37852, DEL-6202)
+
+- Updated legacy delegate images `kubectl` version to 1.25.8. (PL-38281, DEL-6087)
+
+#### Continuous Delivery & GitOps
+
+- Release notes candidate set to Yes and No. (CDS-59026)
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:43]: not sure if you meant to leave this placeholder

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 4]
```diff
@@ -14,6 +14,76 @@ For Harness SaaS release notes, see [Harness SaaS Release Notes](https://develop
 
 Release notes are displayed with the most recent release first.
 
+### June 13, 2023, version 79230
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:17]:
```suggestion
## June 13, 2023, version 79230
```

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 5]
```diff
@@ -14,6 +14,76 @@ For Harness SaaS release notes, see [Harness SaaS Release Notes](https://develop
...
+
+#### Harness Delegate
+
+- Shell delegates did not upgrade automatically. Delegate URLs were not included in installation scripts. (PL-38304, DEL-6276)
+
+   This issue is fixed with a code enhancement.
+
+- Updated the error message for failed task execution to include the delegate host name or ID. (PL-38329, DEL-6187)
+
 ### April 26, 2023, version 78926
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:87]:
```suggestion
## April 26, 2023, version 78926
```

File: release-notes/self-managed-enterprise-edition.md [hunk 6]
```diff
@@ -14,7 +14,665 @@ import delete_project from './static/delete-project.png'
...
+| CI Manager | 3621 |
+| Pipeline Service | 1.26.10 |
+| Platform Service | 78903 |
+| Access Control Service | 78705 |
+| Change Data Capture | 79230 |
+| Test Intelligence Service | release-172 |
+| NG UI | 0.347.19 |
+| LE NG | 67808 |
+
+### Self-Managed Enterprise Edition
```
Comment (aimurphy) [inline release-notes/self-managed-enterprise-edition.md:39]: Due to the way the minitoc works, you might want to use level 4 headings for these.

My rule of thumb for headings on tabs is to use level 3 only if the headings are on the `default` tab.  If multiple tabs have headings, it's better to use level 4 so there are no dead links on the minitoc.

Alternately, you could use tabs within tabs to eliminate the need for headings.

**Review by aimurphy — COMMENTED** [review]
A couple items that need to be addressed (placeholder text + heading level)

**Review by aimurphy — APPROVED** [review]

File: release-notes/self-managed-enterprise-edition.md (hunk 6 above)
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md]: Thanks. I updated to use level 4.

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**